evap/staff/forms.py

```diff
@@ -336,14 +343,17 @@ def save(self, *args, **kw):
         single_result_questionnaire = Questionnaire.single_result_questionnaire()
         single_result_question = single_result_questionnaire.questions.first()
 
-        contribution, created = Contribution.objects.get_or_create(evaluation=self.instance, responsible=True, can_edit=True, textanswer_visibility=Contribution.GENERAL_TEXTANSWERS)
+        contribution, created = Contribution.objects.get_or_create(evaluation=self.instance, can_edit=True, textanswer_visibility=Contribution.GENERAL_TEXTANSWERS)
         contribution.contributor = self.cleaned_data['responsible']
```

Comment:
So for single results, we will have a contribution where the responsible is the contributor. Is this what you intended?

Reply:
yes, that's what we had so far. as it doesn't make too much sense now that we don't need a contribution to have a responsible, this will be changed. i already have this prepared somewhere and will add it to this commit.

Reply:
added a new commit for single result contribution changes